server/py/framework/db/sqldb/db.py

```diff
@@ -3185,6 +3203,88 @@ def _calculate_runs_counters(
             project_to_running_runs_count,
         )
 
+    def _calculate_alert_activations_counters(
+        self,
+        session,
+        projects_with_creation_time: list[tuple[str, datetime]],
+    ) -> tuple[
+        dict[str, int],
+        dict[str, int],
+        dict[str, int],
+    ]:
+        project_to_endpoint_alerts_count = collections.defaultdict(int)
+        project_to_job_alerts_count = collections.defaultdict(int)
+        project_to_other_alerts_count = collections.defaultdict(int)
+
+        project_filter_conditions = self._generate_project_filter_conditions(
+            projects_with_creation_time
+        )
+
+        last_day = mlrun.utils.datetime_now() - timedelta(hours=24)
+
+        query_results = (
+            session.query(
+                AlertActivation.project,
+                func.count(
+                    case(
+                        (
+                            AlertActivation.entity_kind
+                            == mlrun.common.schemas.alert.EventEntityKind.MODEL_ENDPOINT_RESULT,
+                            1,
+                        )
+                    )
+                ).label("model_endpoint_alerts_count"),
+                func.count(
+                    case(
+                        (
+                            AlertActivation.entity_kind
+                            == mlrun.common.schemas.alert.EventEntityKind.JOB,
+                            1,
+                        )
+                    )
+                ).label("job_alerts_count"),
+                func.count(
+                    case(
+                        (
+                            ~AlertActivation.entity_kind.in_(
```

Comment:
While this is correct, I think it's clearer using 
```suggestion
                            AlertActivation.entity_kind.not_in(
```